otx/algorithms/segmentation/configs/base/data/data_pipeline.py

```diff
@@ -37,12 +36,13 @@
     ),
     dict(type="Normalize", **__img_norm_cfg),
     dict(type="Pad", size=__crop_size, pad_val=0, seg_pad_val=255),
-    dict(type="RandomRotate", prob=0.5, degree=30, pad_val=0, seg_pad_val=255),
     dict(type="DefaultFormatBundle"),
     dict(type="Collect", keys=["img", "gt_semantic_seg"]),
 ]
+
 test_pipeline = [
     dict(type="LoadImageFromFile"),
+    dict(type="LoadFromFiles", flag=False),
```

Comment:
Could we remove this line and then edit the `LoadImageFromFile`?

It seems to confuse users since `LoadImageFromFile` and `LoadFromFiles` looks very similar.

```python3
dict(type="LoadImageFromFile", use_otx_adapter=False)
```

Reply:
Hmmm, yes, I can add this to LoadImageFromFile thank you. My solution is a bad workaround, I agree